deploy/helm/grafana-operator/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,45 @@
+{{- if.Values.serviceMonitor.enabled -}}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ include "grafana-operator.fullname" . }}
+  labels:
+    {{- include "grafana-operator.labels" . | nindent 4 }}
+    {{- with .Values.additionalLabels }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
+    {{- with .Values.serviceMonitor.additionalLabels }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
```

Comment:
Due to earlier PR we set namespace in all templates, so please add that to this one as well.